src/index.md

```diff
@@ -0,0 +1,23 @@
+# Welcome to Luos documentation
+
+## Introduction 
+
+We started designing Luos with the conviction that building electronic systems should be made easier than it is today. Most of the time should be spent on designing the applications and behaviors instead of on complex and time-and-money-eating technicalities. To give a simple example, adding a new sensor &mdash;for instance a distance sensor&mdash; to an electronic device in conception should not take more than a few minutes. So you can try, test and iterate fast on a project to truly design what users want.
+
+**Luos works like <a href="https://en.wikipedia.org/wiki/Microservices" target="_blank">microservices architecture</a> in the software world, and a <a href="https://en.wikipedia.org/wiki/Distributed_operating_system" target="_blank">distributed operating systems</a>: it encapsulates any software or hardware function to make it communicate and work with any other encapsulated module, however it was developed, either on bare metal or on top of an embedded OS.**
+Watch this video for additional details:
+
+<iframe width="800" height="450" src="https://www.youtube.com/watch?v=xQe3z0M_FE8?feature=oembed" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe><br />
+
+**You are not familiar with Luos operation?**
+ ->  Start reading the [General use](./_pages/first_steps/general-use.md) page.
+ 
+**You want to make your own board with [Luos drivers](./_pages/modules/drivers.md) or [Luos apps](./_pages/modules/apps.md)?**
+ -> Start reading how to integrate Luos in you [Development environment](./_pages/integrating_luos/dev-env.md).
+ -> Then learn how to [Create and initialize a module](./_pages/modules/create-init.md).
+
```

Comment:
```suggestion
**You want to use [pre-maid board](!!!!There is no such a page Planned!!!!!) and [Luos drivers](./_pages/modules/drivers.md) ?**
 -> You can make your own embedded [Luos apps](./_pages/modules/apps.md).
 -> You can control your devices trough a [Gate](./_pages/modules/modules_list/gate.md) modules using [Pyluos](./_pages/first_steps/pyluos.md).
```

Reply:
whatch out in Summary.md **Boards list** is linked to **Button** page. We could make a specific page listing boards. In my suggestion I can't find such a page so I put a lot of !!!!! instead...